### DISCUSSION TITLE
Step 2: Running Stack Preprocessing

### DISCUSSION THREAD

**Discussion by lumiso** [BrainEnergyLab/Inflammation-Index#23] [Fiji Plugin Issues]
Good afternoon. 
My work stops at Step 2: Running Stack Preprocessing.
 It says that the .ini file is incorrectly formatted, although it is completely identical to the one in the sample with the same localization. 
Can you please tell me how to fix this? 
Sincerely,
LuMiSo

**Comment by DKClarke** [comment]
Hi LuMiSo, sorry to hear you're having trouble. Could you either send me a link to your ini file, or copy and paste the content in a message here, and I'll try it on my end to see if I can figure out what is going wrong.

**Comment by lumiso** [comment]
Fiji MY.app.rar
<https://drive.google.com/file/d/10vHzDvjBRcUX8VTVkqLbEAsYAQDnrmoI/view?usp=drive_web>
 try_8.rar
<https://drive.google.com/file/d/12wNt90lCKDQwCKpphkDAk_miVV4_7dAt/view?usp=drive_web>
Hello Devin
I solved the issue with the ini file thanks to your samples. Now I can go
to the end, but at the penultimate stage there is no Scholl analysis. Could
you see what is wrong. I am attaching the archive of the program and the
processed file. There is also a screenshot of the Scholl analysis error.
Thanks in advance
LuMiSo

22.12.2022 16:23, Devin Clarke ***@***.***>
>Hi LuMiSo, sorry to hear you're having trouble. Could you either send me a
link to your ini file, or copy and paste the content in a message here, and
I'll try it on my end to see if I can figure out what is going wrong.
>
> --
> Reply to this email directly or view it on GitHub:
>
https://github.com/BrainEnergyLab/Inflammation-Index/discussions/23#discussioncomment-4476127
> You are receiving this because you authored the thread.
>
> Message ID:
<BrainEnergyLab/Inflammation-Index/repo-discussions/23/comments/
***@***.***>

**Reply by DKClarke** [reply]
@lumiso I just sent a request to access to your gdrive files

**Comment by lumiso** [comment]
Hello

I sent a reply from a different address because no files were uploaded from that one

LuMiSo


22.12.2022 16:23, Devin Clarke ***@***.***>
>Hi LuMiSo, sorry to hear you're having trouble. Could you either send me a link to your ini file, or copy and paste the content in a message here, and I'll try it on my end to see if I can figure out what is going wrong. 
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/BrainEnergyLab/Inflammation-Index/discussions/23#discussioncomment-4476127
> You are receiving this because you authored the thread.
> 
> Message ID: ***@***.***>


-- реклама -----------------------------------------------------------
Поторопись зарегистрировать самый короткий почтовый адрес @i.ua
https://mail.i.ua/reg - и получи 1Gb для хранения писем

**Comment by lumiso** [comment]
I gave permission

23.12.2022 14:02, Devin Clarke ***@***.***>
>@lumiso I just sent a request to access to your gdrive files
> 
> 
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/BrainEnergyLab/Inflammation-Index/discussions/23#discussioncomment-4484223
> You are receiving this because you were mentioned.
> 
> Message ID: ***@***.***>


-- реклама -----------------------------------------------------------
Поторопись зарегистрировать самый короткий почтовый адрес @i.ua
https://mail.i.ua/reg - и получи 1Gb для хранения писем

**Reply by DKClarke** [reply]
Hi @lumiso I've just uploaded a new version of the plugin [here](https://github.com/BrainEnergyLab/Inflammation-Index/tree/master/Microglia%20Morphology%20Analysis%20Plugin%20-%20ImageJ). It should solve the issue you were having, please let me know. I'm working on fixing the issue with the .ini file now as well.

**Comment by lumiso** [comment]
Dear Devin
Thanks for the help. The error disappeared, but the result of the Scholl analysis did not appear. I send what happened with screenshots and text from the log file from ***@***.*** Sincerely, LiuMiSo

28.12.2022 13:51, Devin Clarke ***@***.***>
>Hi @lumiso I've just uploaded a new version of the plugin [here](https://github.com/BrainEnergyLab/Inflammation-Index/tree/master/Microglia%20Morphology%20Analysis%20Plugin%20-%20ImageJ). It should solve the issue you were having, please let me know. I'm working on fixing the issue with the .ini file now as well.
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/BrainEnergyLab/Inflammation-Index/discussions/23#discussioncomment-4507982
> You are receiving this because you were mentioned.
> 
> Message ID: ***@***.***>


-- реклама -----------------------------------------------------------
Поторопись зарегистрировать самый короткий почтовый адрес @i.ua
https://mail.i.ua/reg - и получи 1Gb для хранения писем

**Reply by mkmarshall0** [reply]
Hello, I am having the same issue with the Sholl analysis - did you ever find a solution to this?
